code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -138,10 +139,16 @@
 
 /obj/item/reagent_containers/hypospray/medipen/examine()
 	. = ..()
-	if(reagents?.reagent_list.len)
-		. += span_notice("It is currently loaded.")
+	if (old_text)     //if old_text = true have the old examine text, if otherwise have the new examine text
+		if (length(reagents?.reagent_list))
+			. += span_notice("It is loaded.")
+		else
+			. += span_notice("It is spent.")
 	else
-		. += span_notice("It is spent.")
+		if(length(reagents?.reagent_list))
+			. += span_notice("There\'s a small LCD screen on the side of the medipen which reads, 'WARNING: This medipen contains [pretty_string_from_reagent_list(reagents.reagent_list, TRUE, ", ", TRUE, TRUE)]. Do not use if allergic to any listed chemicals.' in small text.") //cheapskates at nanotrasen couldnt ball out for LED
```

Comment:
Instead of using the exact reagents list, this should instead be generated in `Initialize` and represent what the default state of the reagents are. You can still have the "It is spent" text, but I don't think it should be dynamic, especially for something like CMO hypospray.

Reply:
Hypospray and combat pens don't get the new text

Reply:
I think they ought to since it's nice to be able to tell people hyposprays have omnizine and such through the same system. I just don't think it should auto update.

Reply:
I think they shouldnt because as stated in the past iteration of this PR, "reduces espionage opportunity", you cant put custom reagents in a medipen, you can put custom reagents in a hypospray.

Reply:
But that's why I made the suggestion:

> Instead of using the exact reagents list, this should instead be generated in `Initialize` and represent what the default state of the reagents are. You can still have the "It is spent" text, but I don't think it should be dynamic, especially for something like CMO hypospray.

Reply:
Hypospray bottles are marked and labelled already, there wouldnt be any purpose in having the initialize chemicals displayed, as they would just say "Airuri" or whatever the default chemical in the hypospray is, when it could easily be deducted that Airuri is inside of the hypospray, just by ejecting the cartridge.